staging/src/k8s.io/apiserver/pkg/authorization/cel/matcher.go

```diff
@@ -89,3 +99,18 @@ func (c *CELMatcher) Eval(ctx context.Context, r *authorizationv1.SubjectAccessR
 	// return ALL matchConditions evaluate to TRUE successfully without error
 	return true, nil
 }
+
+func getSpecSchema(r *authorizationv1.SubjectAccessReview) (*spec.Schema, error) {
```

Comment:
While this works in the sense of passing the unit tests, I don't think this is the correct way of obtaining the OpenAPI schema for the SubjectAccessReviewSpec. How is it done correctly?